pkg/cloudprovider/providers/aws/aws.go

```
@@ -1510,16 +1518,16 @@ func (c *Cloud) CreateDisk(volumeOptions *VolumeOptions) (string, error) {
 
 	case VolumeTypeIO1:
 		// See http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html for IOPS constraints
-		if volumeOptions.IOPSPerGB <= 0 || volumeOptions.IOPSPerGB > 30 {
-			return "", fmt.Errorf("invalid iopsPerGB value %d, must be 0 < IOPSPerGB <= 30", volumeOptions.IOPSPerGB)
+		if volumeOptions.IOPSPerGB <= 0 || volumeOptions.IOPSPerGB > MaxIOPSPerGB {
+			return "", fmt.Errorf("invalid iopsPerGB value %d, must be 0 < IOPSPerGB <= %d", volumeOptions.IOPSPerGB, MaxIOPSPerGB)
 		}
 		createType = volumeOptions.VolumeType
 		iops = int64(volumeOptions.CapacityGB * volumeOptions.IOPSPerGB)
-		if iops < 100 {
-			iops = 100
+		if iops < MinTotalIOPS {
+			iops = MinTotalIOPS
```

Is it better to return error message instead of silently capping it?